### ISSUE TITLE
Please archive this repo

### ISSUE THREAD

**Issue by martinvw** [spotify/docker-kafka#104]
Dear @mattnworb & @mbruggmann could it make sense to archive this repo to make it more clear that it is no longer maintained and does not work with current versions of Kafka (clients).

See also: https://github.com/spotify/docker-kafka/issues/102

Thanks a lot for your consideration

**Comment by mbruggmann** [comment]
Yes, I think so. @mattnworb let me know if you disagree, otherwise I'll press the button!

**Comment by mattnworb** [comment]
👍
